tensorflow/cc/gradients/resource_variable_grad_test.cc

```diff
@@ -33,6 +33,7 @@ namespace {
 
 TEST(ResourceVariableGradTest, ReadVariableOpGrad) {
   TensorShape shape({});
+  auto scope_ = Scope::NewRootScope();
```

Comment:
Why the trailing underscore? That's typically for class members, but this is a local variable.

Reply:
Since it was a class member and I didn't rename it.  I'll do so.